report_templates/diffmeth.Rmd.in

```diff
@@ -1,419 +1,1439 @@
 ---
-title: "PiGx: BS-seq"
-output:
-  html_notebook:
-    toc:        TRUE
-    toc_float:  TRUE
-    theme:      "lumen"
-    number_sections: FALSE
-    code_folding: "hide"
-    self_contained: TRUE
-    includes:
-      in_header: pigx_bsseq_logo.html
+title: "PiGx-BSseq: Differential Methylation Report"
+author: "BIMSB Bioinformatics Platform"
 date: '`r format(as.POSIXct(if ("" != Sys.getenv("SOURCE_DATE_EPOCH")) { as.numeric(Sys.getenv("SOURCE_DATE_EPOCH")) } else { Sys.time() }, origin="1970-01-01"), "%Y-%m-%d %H:%M:%S")`'
-bibliography: reports.bib
 link-citations: yes
+output:
+  html_notebook:
+      toc : TRUE
+      depth : 2
+      toc_float : TRUE
+      theme : 'lumen'
+      number_sections : TRUE
+      code_folding : "hide"
+      self_contained : TRUE
+      bibliography : 'report_templates/reports.bib'
 params:
-  source_dir: ''
-  scripts_dir: ''
-  diffmeth_dir: ''
-  genome_dir: ''
-  out_dir:    ''
-  cpgIsland_bedfile:  ''
-  refGenes_bedfile:   ''
-  chrom_seqlengths: ''
-  assembly: ''
-  treatment: ''
-  qvalue: 0.01
-  difference: 25
-  methylDiff_files: list()
-  webfetch:     ''
-
-  methylDiff_file:          ''
-  methylDiff_hyper_file:    ''
-  methylDiff_hypo_file:     ''
-  methylDiff_nonsig_file:   ''
-  methylDiffBed:            ''
-
-  AnnotateDiffMeth: TRUE
-  DiffMeth: TRUE
-  MethCall: TRUE
+  scripts_dir:            ''
+  cpgIsland_bedfile:      ''
+  refGenes_bedfile:       ''
+  chrom_seqlengths:       ''
+  qvalue:                 0.01
+  difference:             25
+  webfetch:               ''
+
+  sampleids:              ''
+  treatments:             ''
+  assembly:               ''
+  context:                ''
+  destranded:             ''
+
+  control_group:          ''
+  treatment_group:        ''
+
+  methylDiff_file:        ''
+  methylBase_file:	      ''
+  methylDiff_results_file: ''
+
+  prefix:                 ''
+  workdir:                ''
+  logo:                   ''
+  cores:                  8
 ---
 
+
+<style>
+#logo
+{
+    position: relative;
+}
+#logo img {
+    /*position: relative;*/
+    top: 25px;
+    /*right: 0px;*/
+    left: 50px;
+    position: fixed;
+    width: 125px;
+    }
+body
+{
+    position: absolute;
+    top: 150px;
+}
```

Comment:
might be better to avoid absolute positioning and add margin-top: 150px instead.